includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -433,6 +433,7 @@ public function remove_invalid_attribute( $element, $attribute, $validation_erro
 		$should_remove = $this->should_sanitize_validation_error( $validation_error, compact( 'node' ) );
 		if ( $should_remove ) {
 			$element->removeAttributeNode( $node );
+			$this->clean_up_after_attribute_removal( $element, $node, $validation_error );
```

Comment:
Placing this here ended up causing an exception in some cases. See https://github.com/ampproject/amp-wp/pull/3682